packages/components/molecules/f-user-message/CHANGELOG.md

```diff
@@ -7,6 +7,13 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 Latest (add to next release)
```

Comment:
Could you please move version to this line instead of "Latest (add to next release)"

Reply:
Fixed that ta